test/Interop/Cxx/namespace/inline-namespace-ambiguity-error.swift

```diff
@@ -0,0 +1,29 @@
+// RUN: rm -rf %t
+// RUN: split-file %s %t
+// RUN: %target-swift-frontend -typecheck -verify -verify-ignore-unknown -I %t/Inputs  %t/test.swift  -enable-cxx-interop
```

Comment:
(Assuming I'm reading this right...) I suppose it's OK to keep this test, but to be honest, I'm not sure it provides a lot of value. In this test suite we don't want to verify that clang correctly diagnoses errors. We could have a test that we correctly propagate clang diagnostics in general, but we don't need to verify each diagnostic. That is really the job of the clang test suite.

Reply:
That's not a clang error, that's a swift error.

Reply:
Ah, OK. In that case ignore this comment :)